source/administration/config-settings.rst

```diff
@@ -1270,11 +1270,11 @@ User sessions are cleared when a user tries to log in. Additionally, a job runs
 Extend session length with activity
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Improves user experience by extending sessions and keeping users logged in if they are active in their Mattermost apps. 
+Improves user experience by extending sessions. Keeps users logged in if they are actively using their Mattermost apps. As of Mattermost v5.26, the Mattermost server sends session expiry push notifications if sessions expire on mobile.
 
-**True**: Sessions will be automatically extended when the user is active in their Mattermost client. User sessions will only expire if they are not active in their Mattermost client for the entire duration of the session lengths defined in the fields below.
+**True**: Sessions will be automatically extended when the user is actively using their Mattermost app. User sessions will only expire if they are not actively using their Mattermost app for the full duration of the session lengths defined in the fields below. When a session expires on the mobile client, the Mattermost server sends mobile devices a session expiry push notification.
 
-**False**: Sessions will not extend with activity in Mattermost. User sessions will immediately expire at the end of the session length or idle timeouts defined below.
+**False**: Sessions will not extend with activity in Mattermost. User sessions will immediately expire at the end of the session length or idle timeouts defined below. When a session expires on the mobile client, no session expiry push notification is sent from the Mattermost server.
```

Comment:
This is not actually accurate afaik. The server still sends it if the setting is false, we just didn't have it when the setting was true until that ticket was completed. TBH I don't think this [ticket ](https://mattermost.atlassian.net/browse/MM-25394)needs docs changes, the ticket just makes the feature work as it was speced. There's no need to call out this specifically in my mind. 

I'd propose closing this PR unless we want to make any of the grammatical changes

Reply:
Sounds good, thanks! Closing this PR.